DEPS

```diff
@@ -606,6 +613,29 @@ hooks = [
       'src/third_party/dart/third_party/7zip.tar.gz.sha1',
     ],
   },
+  {
+    'name': 'emsdk install',
+    'pattern': '.',
+    'condition': 'download_emsdk',
+    'action': [
+      'python',
+      'src/third_party/emsdk/emsdk.py',
```

Comment:
Maybe you want to control the version of python you are using but `emsdk.sh` is the recommended entry point.

Reply:
This needs to run on windows as well as Mac and Linux.  Plus gclient is already python